Traceback (most recent call last):
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/main.py", line 663, in <module>
    main()
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/main.py", line 631, in main
    loss = criterion(outputs, labels)
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/main.py", line 516, in forward
    targets_one_hot = torch.eye(self.weights.size(0))[targets].permute(0, 3, 1, 2).to(targets.device)
RuntimeError: indices should be either on cpu or on the same device as the indexed tensor (cpu)
srun: error: linux45: task 0: Exited with exit code 1
